doc/source/guidelines/PyMAPDL_WSL_Guide.rst

```diff
@@ -0,0 +1,423 @@
+  .. _ref_guide_wsl:
+
+
+ANSYS on Windows Subsystem for Linux and Docker
+###############################################
+
+This guide shows you how to use Ansys products, more specifically PyMAPDL, in the Windows Subsystem for Linux (WSL).
+WSL is a compatibility layer for running Linux binary executables natively on Windows 10, Windows 11, and Windows Server 2019 
+(`Wikipedia-WSL <https://en.wikipedia.org/wiki/Windows_Subsystem_for_Linux>`_).
+
+This guide walks you through the installation of WSL in Windows, and shows how to use it together with ANSYS, PyMAPDL and docker.
+
+For more information about WSL please visit `What is the Windows Subsystem for Linux? <https://docs.microsoft.com/en-us/windows/wsl/about>`_. 
+
+.. warning:: **Disclaimer:** This guide is still in Alpha state. Please proceed with caution. 
+
+
+.. warning:: This guide hasn't been tested with a VPN connection.
+
+
+Running ANSYS on WSL 
+*********************
+
+Install WSL2
+=============
+
+Follow the instructions in the official Microsoft website `Install WSL <https://docs.microsoft.com/en-us/windows/wsl/install/>`_ .
```

Comment:
```suggestion
Follow the `Install WSL <https://docs.microsoft.com/en-us/windows/wsl/install/>`_ on the official Microsoft website.
```

Reply:
I really need to get my head around ``See``...